src/gui/attributetable/qgsorganizetablecolumnsdialog.h

```diff
@@ -76,6 +76,11 @@ class GUI_EXPORT QgsOrganizeTableColumnsDialog : public QDialog, private Ui::Qgs
      */
     void hideAll();
 
+    /**
+     * toggleSelection Toggle the check state of selected fields to hide or show them in the attribute table
```

Comment:
```suggestion
     * Toggle the check state of selected fields to hide or show them in the attribute table
     * \since QGIS 3.36
```

No need to repeat the function name even if it's done for the method above